packages/uikit/src/widgets/Menu/components/UserMenu/index.tsx

```diff
@@ -79,11 +79,11 @@ const UserMenu: React.FC<UserMenuProps> = ({
   const [isOpen, setIsOpen] = useState(false);
   const [targetRef, setTargetRef] = useState<HTMLDivElement | null>(null);
   const [tooltipRef, setTooltipRef] = useState<HTMLDivElement | null>(null);
-  const accountEllipsis = account
-    ? isSidName
-      ? account
-      : `${account.substring(0, 2)}...${account.substring(account.length - 4)}`
-    : null;
+
+  const isNeedEllipsis = account && !isSidName;
+  const accountEllipsis = account ? `${account.substring(0, 2)}...${account.substring(account.length - 4)}` : null;
+  const accountSID = account && isSidName ? account : null;
```

Comment:
uikit doesn't need to know isSidName
it also let network switcher not having text